priv/repo/migrations/20201210043230_create_notes.exs

```diff
@@ -0,0 +1,19 @@
+defmodule ChatApi.Repo.Migrations.CreateNotes do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :body, :text
```

Comment:
sorry i missed this earlier -- let's add `null: false` to all the required fields, e.g.
```
add :body, :text, null: false
```